scripts/common.psm1

```diff
@@ -19,7 +19,7 @@ function Write-Log
         if (($null -ne $TraceMessage) -and ![System.String]::IsNullOrEmpty($TraceMessage)) {
             $timestamp = (Get-Date).ToString('HH:mm:ss')
             Write-Host "[$timestamp] :: $TraceMessage"-ForegroundColor $ForegroundColor
-            Write-Output "[$timestamp] :: $TraceMessage" | Out-File "$PSScriptRoot\$LogFileName" -Append
+            Write-Output "[$timestamp] :: $TraceMessage" | Out-File "$env:TEMP\$LogFileName" -Append
```

Comment:
@shankarseal does this break existing code that gathers logs?

Reply:
Yes I think line 254 of `config_test_vm.psm1` would need to be changed.

Reply:
Updated

Reply:
@shankarseal verified that testlog.log is in the artifacts now, please re-review